### DISCUSSION TITLE
How to focusout a entry

### DISCUSSION THREAD

**Discussion by zSynctic** [TomSchimansky/CustomTkinter#1090] [Q&A] [answered]
I want to make it that if I press on the entry then type and then press out of the entry on anything like the frame it focuses out

**Comment by Akascape** [comment, marked as answer]
@zSynctic You can use this method to focus out any widget entry:

```python
root.bind("<1>", lambda event: event.widget.focus_set())
```

**Reply by zSynctic** [reply]
by root do you mean the entry or the customtkinter.cTk

**Reply by Akascape** [reply]
@zSynctic root here means the main window. (customtkinter.CTk())
Hope it will help. Don't forget to mark it as answer.

**Reply by zSynctic** [reply]
Oh ok but how can I do it when I'm using classes so for example it's like this class App(customtkinter.CTk)

**Reply by Akascape** [reply]
@zSynctic replace root with self
```python
self.bind("<1>", lambda event: event.widget.focus_set())
```
